.github/workflows/gemini-scheduled-issue-triage.yml

```diff
@@ -14,11 +14,8 @@ defaults:
     shell: 'bash'
 
 permissions:
-  contents: 'read'
   id-token: 'write'
   issues: 'write'
-  statuses: 'write'
-  packages: 'read'
```

Comment:
Need the package read permission to pull mcp server container image required by the issue deduplication workflows